app/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -630,10 +635,14 @@ void setUserName(final String userName) {
   private Collection<EnodeURL> staticNodes;
   private BesuController besuController;
   private BesuConfigurationImpl pluginCommonConfiguration;
+  private Runner runner;
 
   private Vertx vertx;
   private EnodeDnsConfiguration enodeDnsConfiguration;
   private KeyValueStorageProvider keyValueStorageProvider;
+  private final long ephemeryCycle = TimeUnit.DAYS.toSeconds(28);
+  private ScheduledExecutorService ephemeryService;
```

Comment:
Hi! I'm currently learning about Besu and came across your PR as part of my educational exploration. Please feel free to ignore my comment if it's not relevant 😊

I looked into how [ScheduledExecutorService](https://github.com/search?q=repo%3Ahyperledger%2Fbesu%20ScheduledExecutorService&type=code) is used throughout the project, and I noticed that names like `scheduler` or `timerExecutor` are commonly used. In my opinion, one of those might be more suitable here, especially since the instance could potentially be reused for other timer-related functionality in BesuCommand.